code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm

```diff
@@ -90,3 +90,31 @@
 		user << "<span class='warning'>Turn \the [src] off first!</span>"
 		return -1
 	. = ..()
+
+/obj/machinery/rust/gyrotron/verb/rotate_cw()
+	set name = "Rotate (Clockwise)"
+	set src in oview(1)
+	set category = "Object"
+
+	if(usr.restrained() || usr.stat || usr.weakened || usr.stunned || usr.paralysis || usr.resting || !Adjacent(usr))
+		return
+
+	if(anchored)
+		usr << "<span class='notify'>\the [src] is anchored to the floor!</span>"
+		return
+
+	dir = turn(dir, -90)
```

Comment:
Wont a -90 degree turn make it go counter clockwise? Are we looking at this from a compass rose perspective or a grid?